webapp/loggedin/surveyViews/nodeDefsSelector/components/entitySelector.js

```diff
@@ -8,14 +8,14 @@ import { nbsp } from '@core/stringUtils'
 
 import Dropdown from '@webapp/commonComponents/form/dropdown'
 
-const getEntities = (hierarchy, lang) => {
+const _generateDropdownItems = (hierarchy, lang) => {
   const entities = []
 
   const traverse = (nodeDef, depth) => {
     const label = NodeDef.getLabel(nodeDef, lang)
     entities.push({
       key: NodeDef.getUuid(nodeDef),
-      value: nbsp + R.repeat(nbsp + nbsp, depth).join('') + label,
+      value: `${nbsp}${R.repeat(nbsp + nbsp, depth).join('')}${label}${NodeDef.isAnalysis(nodeDef) ? ' (V)' : ''}`,
```

Comment:
I have seen adding the (V) also in Survey.getHierarchy. can we add this instead only once to NodeDef.getLabel?

Reply:
ok